app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<table id="index_task_table">
+  <% tasks = Task.all %>
+    <tr>
+      <th>#</th>
+      <th>TASK</th>
+      <th>ACTIONS</th>
+     <%#  <th>DESCRIPTION</th>
+      <th>DUE DATE</th>
+      <th>COMPLETED ON</th>%>
+    </tr>
+    <tbody>
+    <% ordered_tasks = @tasks.order(:due_date) %>
+  <% ordered_tasks.each_with_index do |task, i| %>
+      <tr>
+        <td><%=i + 1%></td>
+        <% if task.completion_date.nil? %>
+          <td><a href="/tasks/<%= task.id %>"><%= task.item %> 🔍 </a></td>
```

Comment:
You should use `link_to` and you should use a named path like `task_path(task.id)`